test/e2e/webhook_test.go

```diff
@@ -1076,10 +1076,10 @@ var _ = Describe("webhook tests for ResourceOverride CREATE operations resource
 
 	It("should deny create RO with invalid resource override", func() {
 		Consistently(func(g Gomega) error {
-			By("create 2nd resourceOverride with same resource selection")
+			By("deny create 2nd resourceOverride with same resource selection")
 			ro1 := &placementv1alpha1.ResourceOverride{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      fmt.Sprintf("test-ro-%d", GinkgoParallelProcess()),
+					Name:      fmt.Sprintf("test-create-ro-%d", GinkgoParallelProcess()),
```

Comment:
i don't think this's the root cause. this create should always fail and ro cannot be created in this case.